src/main/java/gnu/io/RXTXCommDriver.java

```diff
@@ -521,7 +491,7 @@ private void registerScannedPorts(int PortType)
 					useFallback = false;
 				}
 				catch (Throwable ex) {
-					log.debug("Error reading the registry to get port list " + ex.getMessage());					
+					log.debug("Error reading the registry to get port list: {}", ex.getMessage());
```

Comment:
Again, do you have a particular preference for suppressing the stack trace? I can see where there might be some concern about a library being too verbose with its errors.

```suggestion
					log.debug("Error reading the registry to get port list:", ex);
```